### DISCUSSION TITLE
[Question]: chat completion aborts if create new chat

### DISCUSSION THREAD

**Discussion by ad-astra-video** [danny-avila/LibreChat#5122] [Q&A]
### What is your question?

The chat completion request aborts if i create a new chat and send a new prompt.  Is there any way to change this? I would like to have more than one response generating at a time.

### More Details

I am using a vllm server locally and see logs like this when i send a request, then create a new chat to send an additional request:

```
INFO:     172.17.0.1:54662 - "POST /v1/chat/completions HTTP/1.1" 200 OK
INFO 12-26 20:02:41 engine.py:267] Added request chatcmpl-d3e09827e54548e08ed914c0fad97b6a.
INFO 12-26 20:02:43 engine.py:285] Aborted request chatcmpl-d3e09827e54548e08ed914c0fad97b6a.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
no this is intended, you can open new chat in a new window to do this. by default, LibreChat only allows 2 concurrent message streams, but this can be configured: 

https://www.librechat.ai/docs/configuration/mod_system#message-rate-limiting
